client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/step-content.tsx

```diff
@@ -9,11 +13,47 @@ type StepContentProps = {
 	goToStep?: NavigationControls[ 'goToStep' ];
 };
 
-const StepContent = ( { siteSlug, submit, goNext, goToStep }: StepContentProps ) => (
-	<div className="launchpad__content">
-		<Sidebar siteSlug={ siteSlug } submit={ submit } goNext={ goNext } goToStep={ goToStep } />
-		<LaunchpadSitePreview siteSlug={ siteSlug } />
-	</div>
-);
+function sortByRegistrationDate( domainObjectA: ResponseDomain, domainObjectB: ResponseDomain ) {
```

Comment:
To confirm, the idea here is that the most recently registered domain is the one we want. So we fetch all domains a user has registered, and pick the most recent. Seems likely this will work.

I guess there's no way to directly get the domain that was registered during the current onboarding flow? I might have thought that was saved somehow during the earlier onboarding screens, and then accessible to us.

Reply:
> To confirm, the idea here is that the most recently registered domain is the one we want. So we fetch all domains a user has registered, and pick the most recent. Seems likely this will work.

Correct. If the user purchased a domain, then we will use the most recently registered domain.

> I guess there's no way to directly get the domain that was registered during the current onboarding flow? I might have thought that was saved somehow during the earlier onboarding screens, and then accessible to us.

There is a way to pass the domain through the onboarding screens, but that won't solve the site preview issue because we will not be able to load the preview based on that URL/slug if the newly purchased custom domain is not completely resolved (SSL certificates). It would also require more changes across the codebase (change stepper flow redirection URLs and checkout flow redirection if the user cancels out of the checkout process). We figured it would be best to use the domain endpoint to fetch the wp & non wp domains to solve the site preview iframe loading issue and at the same time fix the sidebar URL using the same data.